ios/NativeBridge.swift

```diff
@@ -201,7 +201,11 @@ extension WebAuthError {
             case .pkceNotAllowed: code = "PKCE_NOT_ALLOWED"
             case .idTokenValidationFailed: code = "ID_TOKEN_VALIDATION_FAILED"
             case .other: code = "OTHER"
-            default: code = "UNKNOWN"
+            default: if let cause = self.cause as? AuthenticationError {
```

Comment:
The AuthenticationError will come wrapped in a `WebAuthError` with code `other`, never under `unknown` (`unknown` [does not include a cause](https://auth0.github.io/Auth0.swift/documentation/auth0/webautherror), and that's what differentiates `other` from `unknown`):
<img width="893" alt="Screenshot 2023-09-19 at 11 38 47 AM" src="https://github.com/auth0/react-native-auth0/assets/5055789/7bbc4542-776a-4658-834a-4779161b2caf">

Which means this unwrap will never be executed, because the `other` case is being handled before.
I'd suggest testing this manually.